Buildings/Controls/OBC/ASHRAE/PrimarySystem/ChillerPlant/Staging/Processes/Subsequences/EnableCWPump.mo

```diff
@@ -0,0 +1,209 @@
+within Buildings.Controls.OBC.ASHRAE.PrimarySystem.ChillerPlant.Staging.Processes.Subsequences;
+block EnableCWPump
+  "Generate staging index for condenser water pump control"
+
+  Buildings.Controls.OBC.CDL.Interfaces.BooleanInput uUpsDevSta
+    "Status of resetting status of device before enabling or disabling condenser water pump"
+    annotation (Placement(transformation(extent={{-160,60},{-120,100}}),
+      iconTransformation(extent={{-140,60},{-100,100}})));
+  Buildings.Controls.OBC.CDL.Interfaces.BooleanInput uStaUp
+    "Indicate if there is stage-up command"
+    annotation (Placement(transformation(extent={{-160,20},{-120,60}}),
+      iconTransformation(extent={{-140,0},{-100,40}})));
+  Buildings.Controls.OBC.CDL.Interfaces.BooleanInput uStaDow
+    "Indicate if there is stage-down command"
+    annotation (Placement(transformation(extent={{-160,-60},{-120,-20}}),
+      iconTransformation(extent={{-140,-40},{-100,0}})));
+  Buildings.Controls.OBC.CDL.Interfaces.IntegerInput uChiSta
+    "Current chiller stage index, does not include stages like X + WSE"
+    annotation (Placement(transformation(extent={{-160,-100},{-120,-60}}),
+        iconTransformation(extent={{-140,-100},{-100,-60}})));
+  Buildings.Controls.OBC.CDL.Interfaces.IntegerOutput yChiSta
```

Comment:
This subsequence will not see if or not there is WSE and if or not the WSE is enabled. It will just generate `chiller stage`, as an input to the sequence condenser water pump controller `Buildings.Controls.OBC.ASHRAE.PrimarySystem.ChillerPlant.Pumps.CondenserWater.Controller`. See how I composed it in the staging controller `Buildings.Controls.OBC.ASHRAE.PrimarySystem.ChillerPlant.Staging.Processes.Up`.